core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -409,6 +429,8 @@ primaryExpression
     | qualifiedName '(' ASTERISK ')' filter? over?                                        #functionCall
     | processingMode? qualifiedName '(' (setQuantifier? expression (',' expression)*)?
         (ORDER BY sortItem (',' sortItem)*)? ')' filter? (nullTreatment? over)?           #functionCall
+    | qualifiedName '(' setQuantifier? identifier ',' string  listaggOverflowClause? ')'
```

Comment:
I pretty much assume that a special rule specific for `LISTAGG` would be probably more appropriate within the grammar. Something similar to `TRY_CAST` , but which would translate via a `visitXXX` method to a  `io.trino.sql.tree.FunctionCall` instance.